From 7b41f0c38ab34c9a5eec1a70669d183535e86287 Mon Sep 17 00:00:00 2001
From: Florian Schmaus <flo@geekplace.eu>
Date: Fri, 17 Dec 2021 14:05:50 +0100
Subject: [PATCH] DRAFT shared/att: Use queue_push_tail for
 BT_ATT_OP_READ_BLOB_REQ

As repoorted by Christopher Schramm [1], this is supposed to fix #220.

1: https://github.com/bluez/bluez/issues/220#issuecomment-996669412

Suggested-by: Christopher Schramm <git@cschramm.eu>
---
 src/shared/att.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/shared/att.c b/src/shared/att.c
index 169f726e4..5ce735a07 100644
--- a/src/shared/att.c
+++ b/src/shared/att.c
@@ -1653,7 +1653,6 @@ int bt_att_resend(struct bt_att *att, unsigned int id, uint8_t opcode,
 
 	switch (opcode) {
 	/* Only prepend requests that could be a continuation */
-	case BT_ATT_OP_READ_BLOB_REQ:
 	case BT_ATT_OP_PREP_WRITE_REQ:
 	case BT_ATT_OP_EXEC_WRITE_REQ:
 		result = queue_push_head(att->req_queue, op);
